modules/tinymce/src/plugins/charmap/test/ts/browser/CharmapSearchTest.ts

```diff
@@ -1,11 +1,18 @@
 import { Assertions, Chain, FocusTools, Guard, Keyboard, Keys, Log, Pipeline, UiFinder, Waiter } from '@ephox/agar';
 import { UnitTest } from '@ephox/bedrock-client';
 import { TinyApis, TinyLoader, TinyUi } from '@ephox/mcagar';
+import { PlatformDetection } from '@ephox/sand';
 import { Attribute, SugarBody, SugarElement } from '@ephox/sugar';
 import CharmapPlugin from 'tinymce/plugins/charmap/Plugin';
 import SilverTheme from 'tinymce/themes/silver/Theme';
 
 UnitTest.asynctest('browser.tinymce.plugins.charmap.SearchTest', (success, failure) => {
+  // TODO: TINY-6598: Test is broken on Chromium Edge 86, so we need to investigate
+  const platform = PlatformDetection.detect();
+  if (platform.browser.isChrome() && platform.os.isWindows()) {
```

Comment:
Can platform.browser tell the difference between Chrome and Chromium Edge?

Reply:
No, not currently.